tests/dummy/app/styles/app.scss

```diff
@@ -0,0 +1,7 @@
+table.logs, table.logs th, table.logs td {
```

Comment:
These styles should live in a component-specific CSS file like here: https://github.com/CenterForOpenScience/ember-osf/tree/develop/tests/dummy/app/components/files-tree

https://github.com/CenterForOpenScience/ember-osf/blob/develop/.github/CONTRIBUTING.md#styling-components